lib/rubocop/cop/packaging/require_with_relative_path.rb

```diff
@@ -64,26 +65,39 @@ def on_new_investigation
         def on_send(node)
           return unless require?(node)
 
-          add_offense(node)
+          add_offense(node) do |corrector|
+            corrector.replace(node, good_require_call)
+          end
+        end
+
+        # Called from on_send, this method helps to replace
+        # the "bad" require call with the "good" one.
+        def good_require_call
+          bad_part = @str.match(%r{.*/lib/}).to_s
+          good_call = @str.delete_prefix(bad_part)
+          %(require "#{good_call}")
         end
 
         # This method is called from inside `#def_node_matcher`.
         # It flags an offense if the `require` call is made from
         # anywhere except the "lib" directory.
         def falls_in_lib?(str)
+          @str = str
```

Comment:
I don't like this instance variable name, since it's not helpful to understand the code. But if the instance variable name (`@str`) is not informative, the parameter to the method (/`str`) is not informative either, and you followed the existing pattern already there. So this can be improved in the future :+1:.

Reply:
So the `str` is a token which is a part of the AST, so I can't definitely change that. Now on the other hand, `@str` is just an instance variable that can be named anything but I preferred to keep the same name since they are the same thing!
But fair enough, I've got no problem with changing the name! ^.^

Thanks for the review! 💜